final/app.js

```diff
@@ -0,0 +1,122 @@
+/*
+GAME RULES:
+
+- The game has 2 players, playing in rounds
+- In each turn, a player rolls a dice as many times as he whishes. Each result get added to his ROUND score
+- BUT, if the player rolls a 1, all his ROUND score gets lost. After that, it's the next player's turn
+- The player can choose to 'Hold', which means that his ROUND score gets added to his GLBAL score. After that, it's the next player's turn
+- The first player to reach 100 points on GLOBAL score wins the game
+
+*/
+
+var scores, roundScore, activePlayer, gamePlaying;
+
```

Comment:
Some of these variables are being treated as arrays elsewhere in the code.
In order to help your IDE to better inform you of errors (if using JS 5) then you should declare
the variable as follows:
`
var purpleMonkey = new Array();
`
OR
`
var purpleMonkey = [];
`
N.B From JS2015 onawards you would use the following syntax:
`
let purpleMonkey = new Array();
`